internal/services/zero_trust_tunnel_cloudflared_virtual_network/model.go

```diff
@@ -15,16 +16,26 @@ type ZeroTrustTunnelCloudflaredVirtualNetworkResultEnvelope struct {
 type ZeroTrustTunnelCloudflaredVirtualNetworkModel struct {
 	ID               types.String      `tfsdk:"id" json:"id,computed"`
 	AccountID        types.String      `tfsdk:"account_id" path:"account_id,required"`
-	IsDefault        types.Bool        `tfsdk:"is_default" json:"is_default,optional,no_refresh"`
 	Name             types.String      `tfsdk:"name" json:"name,required"`
 	Comment          types.String      `tfsdk:"comment" json:"comment,optional"`
-	IsDefaultNetwork types.Bool        `tfsdk:"is_default_network" json:"is_default_network,optional"`
+	IsDefaultNetwork types.Bool        `tfsdk:"is_default_network" json:"is_default_network,computed_optional"`
 	CreatedAt        timetypes.RFC3339 `tfsdk:"created_at" json:"created_at,computed" format:"date-time"`
 	DeletedAt        timetypes.RFC3339 `tfsdk:"deleted_at" json:"deleted_at,computed" format:"date-time"`
 }
 
+type newZeroTrustTunnelCloudflaredVirtualNetworkModel struct {
+	Name      types.String `json:"name,required"`
+	Comment   types.String `json:"comment,optional"`
+	IsDefault types.Bool   `json:"is_default,required"`
+}
+
 func (m ZeroTrustTunnelCloudflaredVirtualNetworkModel) MarshalJSON() (data []byte, err error) {
```

Comment:
we shouldn't be modifying the marshaler given it's all codegen'd and makes resolving conflicts far more difficult. instead, if you need an alias, you should manually add the fields to the CRUD operations after the `data = &env.Result` bits to correctly populate the values.